src/main/java/io/temporal/samples/polling/README.md

```diff
@@ -0,0 +1,9 @@
+# Polling
+
+These samples show three different best practices for polling.
+
+1. [Frequent Polling](frequent/README.md)
+2. [Infrequent Polling](infrequent/README.md)
+3. [Periodic Polling](periodic/README.md)
```

Comment:
I think this structure is very confusing. I think it should by "Frequently Polling Activity" "Infrequently Polling Activity" and "Periodic execution on a sequence of activities"